cross-stage.sh

```diff
@@ -32,7 +32,7 @@ ADDITIONAL_PACKAGES="
   dev-build/cmake
   dev-lang/rust
   net-wireless/iw
-  app-misc/screen
+  =app-misc/screen-5.0.0
```

Comment:
why pinning screen?

Reply:
https://bugs.gentoo.org/953896

Reply:
It shouldn't be needed if it is an old version

Reply:
okay removing